libc/src/__support/FPUtil/FPBits.h

```diff
@@ -39,41 +39,41 @@ enum class FPEncoding {
   X86_ExtendedPrecision,
 };
 
-template <FPType> struct FPBaseProperties {};
+template <FPType> struct FPBaseAttr {};
```

Comment:
Let's add a class comment. Something along the lines of "`FPBaseAttr` defines the layout (sign, exponent, mantissa, ...) of a floating point type in memory.".